modules/samples/firealarm/component/analytics/src/main/resources/carbonapps/temperature_sensor/temperature_stream/org.wso2.iot.devices.temperature_1.0.0.json

```diff
@@ -0,0 +1,20 @@
+{
+  "name": "org.wso2.iot.devices.temperature",
+  "version": "1.0.0",
+  "nickName": "Temperature Data",
+  "description": "Temperature data received from the Device",
+  "metaData": [
+    {"name":"owner","type":"STRING"},     
+    {"name":"deviceType","type":"STRING"},
+    {"name":"deviceId","type":"STRING"},  
+    {"name":"time","type":"LONG"}
```

Comment:
Ack, This is a generated code with iot device type maven tooling. Hence needed to fix formatting in the tooling.